scripts/FCCee/ALLEGRO/ALLEGRO_o1_v03/ALLEGRO_o1_v03_script.sh

```diff
@@ -0,0 +1,22 @@
+printf "\n\nSETUP PHASE:\n"
+
+[ -z "$KEY4HEP_STACK" ] && source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh
+
+
+# simulation phase
+printf "\n\nSIM-DIGI-RECO PHASE:\n"
```

Comment:
```suggestion
echo "\n\nSIM-DIGI-RECO PHASE:\n"
```
Use only `echo` for consistency. Same comment for the other occurrences of `printf`.